tests/conftest.py

```diff
@@ -83,8 +88,23 @@ def _get_port() -> int:
         # 1. normal link, 2. debug link and 3. webauthn fake interface
         return 20000 + int(worker_id[2:]) * 3
 
+    # So that we can generate coverage reports
+    profiling = os.environ.get("TREZOR_PROFILING") == "1"
+    if profiling:
+        core_dir = HERE.parent / "core"
+        profiling_wrapper = core_dir / "prof" / "prof.py"
+        main_args = [str(profiling_wrapper)]
+        # So that the coverage reports have the correct paths
+        os.environ["TREZOR_SRC"] = str(core_dir / "src")
+    else:
+        main_args = ["-m", "main"]
```

Comment:
i _think_ we can keep the main_args empty in this case? this `-m main` is a compatibility thing IIRC and we shouldn't be needing it anymore

Reply:
Changed in [7543c1d](https://github.com/trezor/trezor-firmware/pull/2857/commits/7543c1d620a59dd353638d7254e3832d35c8ea12) - works fine